modyn/selector/internal/storage_backend/local/local_storage_backend.py

```diff
@@ -1,27 +1,127 @@
 import logging
-from typing import Any, Iterable, Optional
+from pathlib import Path
+from typing import Iterable, Optional
 
+import numpy as np
+from modyn.common.local_storage_backend import LocalStorageBackend as LocalStorageBackendCPP
 from modyn.selector.internal.storage_backend import AbstractStorageBackend
 
 logger = logging.getLogger(__name__)
 
+MAX_SAMPLES_IN_FILE = 1000000
```

Comment:
should we make this a modyn system parameter?

Reply:
yeah we should - where should I add it?

Reply:
done